### PR TITLE
Fix Postgres array column type support

### PR DESCRIPTION
**Pull request by azelenets**

### CODE REVIEW

**Comment by ldlsegovia** [comment]
@azelenets could you add a test here https://github.com/platanus/activeadmin_addons/blob/master/spec/lib/support/select_helpers_spec.rb#L55?

**Comment by ldlsegovia** [comment]
related with https://github.com/platanus/activeadmin_addons/issues/186

File: spec/lib/support/select_helpers_spec.rb [hunk 1]
```diff
@@ -67,6 +67,25 @@ def self.check_invalid_options(method_name)
       expect(instance.array_to_select_options).to eq(result)
     end
 
+    context 'with array input value' do
+      let(:object) do
+        ObjectStruct = Struct.new(:array_of_integers)
+        ObjectStruct.new([1 , 2])
```
Comment (monkeyci) [inline spec/lib/support/select_helpers_spec.rb]: Space found before comma.

File: spec/lib/support/select_helpers_spec.rb [hunk 2]
```diff
@@ -67,6 +67,25 @@ def self.check_invalid_options(method_name)
...
+      end
+      let(:method) { :array_of_integers }
+
+      it "returns selected items only" do
+        result = [
+          { id: 1, text: 1, selected: "selected" },
+          { id: 2, text: 2, selected: "selected" }
+        ]
+
+        p instance
```
Comment (ldlsegovia) [inline spec/lib/support/select_helpers_spec.rb]: @azelenets could you remove this line please?

**Pull request merged**